packages/web/src/common/store/cache/tracks/utils/fetchAndProcessStems.ts

```diff
@@ -48,14 +48,16 @@ export function* fetchAndProcessStems(trackId: ID) {
     orig_filename: s.orig_filename ?? ''
   }))
 
-  yield* put(
-    cacheActions.update(Kind.TRACKS, [
-      {
-        id: trackId,
-        metadata: {
-          _stems: stemsUpdate
+  if (stemsUpdate.length) {
```

Comment:
What's this for?

Reply:
just drive by... this was causing excess renders